docs/index.md

```diff
@@ -26,3 +30,4 @@ provider "bloxone" {
 
 - `api_key` (String) API key for accessing the BloxOne API. Can also be configured by using the `BLOXONE_API_KEY` environment variable. https://docs.infoblox.com/space/BloxOneCloud/35430405/Configuring+User+API+Keys
 - `csp_url` (String) URL for BloxOne Cloud Services Portal. Can also be configured using the `BLOXONE_CSP_URL` environment variable.
+- `default_tags` (Map of String) Default global tags the client can set for all requests.
```

Comment:
```suggestion
- `default_tags` (Map of String) Tags to default for all resources```

Reply:
Will make the change in provider.go schema